dev-libs/bglibs/bglibs-1.041.ebuild

```diff
@@ -1,51 +0,0 @@
-# Copyright 1999-2018 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=0
-
-inherit fixheadtails toolchain-funcs multilib
-
-DESCRIPTION="Bruce Guenters Libraries Collection"
-HOMEPAGE="http://untroubled.org/bglibs/"
-SRC_URI="http://untroubled.org/bglibs/archive/${P}.tar.gz"
-
-LICENSE="LGPL-2.1"
-SLOT="0"
-KEYWORDS="alpha amd64 ~arm hppa ~mips ppc sparc x86 ~ppc64"
-IUSE=""
-DEPEND=""
-
-src_unpack() {
-	unpack ${A}
-	# disable tests as we want them manually
```

Comment:
you dropped this from the new ebuild. It needs to be kept! The ebuild should ONLY run tests with `FEATURES=test` is passed.  Right now it runs it during src_compile phase:
```
./ltload bg-installer libbg-cli.la libbg.la bg-installer-cli.o -lbg-cli -static 
./ltload selftest-cmp libbg.la  
sh selftests.sh
Creating temporary directory selftests.sh.tmp.10879
Testing crc/gcrc16fwd.c
Testing crc/gcrc16rfl.c
...
```

Reply:
Oh! Thanks.